### ISSUE TITLE
Completely frozen after adding something in a big file

### ISSUE THREAD

**Issue by MonotoneDevelopment** [GraphiteEditor/Graphite#1322]
I was making a Where's Waldo vector when I added one line, and it froze after I added the line. I couldn't do anything, but dropdown menus still showed up and the icons would still light up when the mouse hovered over them. Other than that, I couldn't save my work, move things, select different tools, manipulate nodes, or re-arrange the layers. I spent like 3 hours making Waldo ;-;

**Comment by Keavon** [comment]
So sorry for that painful loss of your hours of creative work and effort :(

We really need to prioritize #929 very soon so that doesn't happen for users anymore.

Just to be clear, it froze but didn't tell you that the editor had crashed, right? I have seen that before but I don't recall if I've attributed it to a specific cause, however it might have resulted from running out of memory. Were you working with purely vector data, or also some bitmap content?

**Comment by MonotoneDevelopment** [comment]
Correct, It did not give any kind of message or anything to show that It had an error, which should probably be a think like a dialog saying " Uh oh, it crashed!" and an option to report the bug in the dialog.

**Comment by Keavon** [comment]
This might be fixed by now. Please reopen if you're able to reproduce it again.
